pyxrf/model/fileio.py

```diff
@@ -1861,3 +1861,98 @@ def export1d(runid, name=None):
     if name is None:
         name = 'scan_'+str(runid)+'.txt'
     t.to_csv(name)
+
+
+def get_header(fname):
+    """
+    helper function to extract header in spec file.
+    .. warning :: This function works fine for spec file format
+    from Canadian light source. Others may need to be tested.
+
+    Parameters
+    ----------
+    fname : spec file name
+    """
+    mydata = []
+    with open(fname, 'r') as f:
+        for v in f:   # iterate the file
+            mydata.append(v)
+            _sign = '#'
+            _sign = _sign.encode('utf-8')
+            if _sign not in v:
+                break
+    header_line = mydata[-2]  # last line is space
+    n = [v.strip() for v in header_line[1:].split('\t') if v.strip()!='']
+    return n
+
+
+def spec_to_hdf(wd, spec_file, spectrum_file, output_file, img_shape,
+                ic_name=None, x_name=None, y_name=None):
+    """
+    Transform spec data to hdf file pyxrf can take. Using this function, users need to
+    have two input files ready, sepc_file and spectrum_file, with explanation as below.
+
+    .. warning :: This function should be better defined to take care spec file in general.
+    The work in suitcase should also be considered. This function works fine for spec file format
+    from Canadian light source. Others may need to be tested.
+
+    Parameters
+    ----------
+    wd : str
+        working directory for spec file, and created hdf
+    spec_file : str
```

Comment:
The spec file in suitcase doesn't work. That's why I wrote a new one, with better interface with pandas. Next step is to think about working on spec in much general way, as I mentioned in warning part. But this is a quick fix, since users are waiting at CLS.